config/packages/doctrine.yaml

```diff
@@ -24,9 +24,9 @@ doctrine:
         naming_strategy: doctrine.orm.naming_strategy.underscore
         auto_mapping: true
         mappings:
-            Domain:
+            Infrastructure:
                 is_bundle: false
                 type: yml
-                dir: '%kernel.project_dir%/config/mapping/orm/query'
-                prefix: 'App\Domain'
-                alias: Domain
+                dir: '%kernel.project_dir%/config/mapping/orm/infrastructure'
```

Comment:
what about `projections` instead `infrastructure `?

Reply:
That is a much better name. I'm going to change it and squash commits